atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -223,6 +169,69 @@ def _send_mail(self, receivers_list, subject, body):
             if s is not None:
                 s.quit()
 
+    def _get_email_from_koji_obj(self, obj):
+        if obj.get('krb_principal') and obj['krb_principal'] != '':
+            return obj['krb_principal'].lower()
+        else:
+            return '@'.join([obj['owner_name'], self.email_domain])
```

Comment:
Oh, actually we want to have exception thrown here - in this case the email would be set to `error_addresses`. If osbs-client has incorrectly configured the plugin then there's not much we can do - error would be logged anyway.

Reset the default value to `None` so that this error would be caught earlier